popup/menu.js

```diff
@@ -51,6 +51,14 @@ bg_port.onMessage.addListener(async (m) => {
             document.getElementById("entity-guest").classList.add("active");
             active = false;
         }
+        if (m.host_version && m.broker_version) {
+            let pvers = chrome.runtime.getManifest().version;
+            let vstr = "v" + pvers;
+            if (m.host_version !== pvers) {
+                vstr += " (host v" + m.host_version + ")";
```

Comment:
Is "host" a term that we established in our user documentation? So, can a user make sense of this?

Also, I'm wondering if it is a feature to hide the native app version when it is synchronized with the extension. Why

Reply:
We use that term multiple times in the README, so I hope our users are aware. If the versions are identical, there is no need to show both, so better avoid cluttering the UI with useless information.